sourcegraph/cody
# Autocomplete: Log wether we're inside a test file

We have code for commands that can be used to estimate if we're inside of a test file or not. Let's log it for Autocomplete so:

- We can estimate CAR for tests
- We can see how much of a pool test completions are

The idea is that we gather some data that we can use to improve this use case. E.g. by force-injecting a different test file to have the LLM infer the right setup or so.

## Test plan

- Enable debug and verbose logging
- Make some completion requests and observe the value of the `testFile` flag.
